Doc/library/sqlite3.rst

```diff
@@ -2266,11 +2333,62 @@ Explanation
 
 .. _sqlite3-controlling-transactions:
 
-Transaction control
-^^^^^^^^^^^^^^^^^^^
+Transaction control via the ``autocommit`` attribute
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the :attr:`~Connection.autocommit` attribute to select
+transaction control mode.
+This attribute should preferrably be set using the *autocommit* parameter
+of :func:`connect`.
+
+Currently, *autocommit* defaults to :const:`DEPRECATED_TRANSACTION_CONTROL`.
+This causes transaction control to be selected using the
+:attr:`~Connection.isolation_level` attribute.
+See :ref:`sqlite3-deprecated-transaction-control` for more information.
+
+Starting in Python 3.14, *autocommit* will default to ``False``,
+which will imply :pep:`249`-compliant transaction control, meaning:
+
+* :mod:`!sqlite3` ensures that a transaction is always open,
+  meaning :meth:`~Connection.commit` and :meth:`~Connection.rollback`
+  will implicitly open new transactions immediately after closing
+  the pending transaction.
+  :mod:`!sqlite3` uses ``BEGIN DEFERRED`` statements when opening transactions.
+* Transactions should be committed explicitly
+  using :meth:`!~Connection.commit`.
```

Comment:
```suggestion
  using :meth:`!commit`.
```

With `!~`, the `~` has no effect (and is displayed instead), while `~!` does work to remove the `!Connection`, but the `!` only works by accident (because there is no `!Connection` class) and the link is still attempted to be resolved and fails, producing a warning. Therefore, this is the least bad option.